Species/Languages.md

```diff
@@ -1,27 +1,117 @@
 ---
-tags: [background, status/needswork]
+tags: [background, status/tim]
 ---
 # Languages 
-***This is a work in progress and details may change as new areas of the world are explored in play. Characters will always have an opportunity to retcon their language choices if elements of this section are revised.***
+Languages in the world of Taelgar are diverse, and yet the binding influence of magic and the gods have kept them from diverging too far apart. 
+## Brief Linguistic History of Taelgar
+The common languages of Taelgar spoken by [[Humans|humans]] and the [[Elder Folk]] all derive from the original languages spoken by the first of each species to awaken. For [[Dwarves|dwarves]], [[Elves|elves]], [[Halflings|halflings]], [[Stoneborn|stoneborn]], and [[Lizardfolk|lizardfolk]] this original language has not changed since the dawn of time. For humans, the original tongue has been split and fractured into different language families over the long history of human settlement in Taelgar.
 
-Languages in the world of Taelgar are diverse, and yet the binding influence of magic and the gods have kept them from diverging too far apart. Nonetheless, individual languages described below typically represent clusters or language families, where knowledge of one member is assumed to represent knowledge of all.
+The history of the languages of other sentient creatures is less well known by scholars, but in most cases, whether due to magic, culture, or other factors, linguistic diversity within species is rare. 
```

Comment:
I think there is no reason not to leave room for goblinoids, at least, to have linguistic diversity, with the hobgoblins in the "main part" of Taelgar mostly speaking the "Cha'mutte's army lingua franca" but leaving the possibility that e.g. hobgoblins in the far east speak a different language. 

But any of the firstborn races or the highly magical races should have a single fixed language, for sure.

Reply:
Maybe just leave this as a background comment or something, and leave the text as written otherwise untouched.

Reply:
Yeah, I was thinking "goblinoids" as the reason for "most" not "all". I'll add a background note, and or just write it up as an exception when I get to goblin languages.